frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -124,10 +129,15 @@ pods:
             template: "{{CONFIG_TEMPLATE_PATH}}/elasticsearch.yml"
             dest: "elasticsearch-{{ELASTIC_VERSION}}/config/elasticsearch.yml"
         readiness-check:
-          cmd: "curl -I -s -f -u {{ELASTICSEARCH_HEALTH_USER}}:{{ELASTICSEARCH_HEALTH_USER_PASSWORD}} localhost:$PORT_HTTP"
+          cmd: "curl -k -I -s -f -u {{ELASTICSEARCH_HEALTH_USER}}:{{ELASTICSEARCH_HEALTH_USER_PASSWORD}} {{ELASTICSEARCH_HEALTH_PROTOCOL}}://$TASK_NAME.$FRAMEWORK_HOST:$PORT_HTTP"
```

Comment:
Is it overly difficult to suck down the ca-bundle? Actually, isn't the ca-bundle visible to the task?

Reply:
I could switch from the OS provided default `curl` to the one that is bundled with DC/OS (`/opt/mesosphere/bin/curl`). The one that is bundled contains DC/OS CA cert bundle. We did the same change for the integration tests.